01-Login/server.js

```diff
@@ -16,20 +15,18 @@ app.set('view engine', 'ejs');
 
 app.use(logger('dev'));
 app.use(express.static(path.join(__dirname, 'public')));
-app.use(express.urlencoded({ extended: false }));
 app.use(express.json());
 
-app.use(session({
-  name: process.env.SESSION_NAME  || 'express-openid-connect',
-  keys: [process.env.SESSION_SECRET || 'Set a SESSION_SECRET value in env'],
-  maxAge: 24 * 60 * 60 * 1000,
-  secure: process.env.NODE_ENV === 'production'
-}));
-
-app.use(auth({
+const config = {
   required: false,
   auth0Logout: true
-}));
+};
+
+if (!config.baseURL && !process.env.BASE_URL && process.env.PORT && process.env.NODE_ENV !== 'production') {
```

Comment:
4 things are being checked here. Consider moving to a variable and giving it a descriptive name